packages/lesswrong/components/sequences/SequencesPage.tsx

```diff
@@ -198,6 +200,11 @@ const SequencesPage = ({ documentId, classes }: {
   const showSequence = useCallback(() => {
     setEdit(false);
   }, []);
+  if (documentId === MARGINAL_FUNDING_SEQUENCE_ID && !currentUser?.isAdmin) {
```

Comment:
I get this when I try to visit this page as a non-admin:
```
Error: Tried to set status to 301 but headers have already been sent with status 200. This may be due to enableResourcePrefetch wrongly being set to true.
    at trySetResponseStatus (/Users/wh/Documents/code/ForumMagnum-vite/packages/lesswrong/server/utils/httpUtil.ts:102:13)
    at <anonymous> (/Users/wh/Documents/code/ForumMagnum-vite/packages/lesswrong/server/apolloServer.ts:514:7)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
```